### ISSUE TITLE
CV does not show plot widget in Jupyter sometimes

### ISSUE THREAD

**Issue by elfwired** [catboost/catboost#347]
Problem: CV does not show plot widget in Jupyter sometimes
catboost version: 0.8.1.1
Operating System: Windows Server 2012
CPU: 4x Intel Xeon v4 16 core (VMWare)
GPU: no GPU
--

When I run `cv` with `plot` option set to `True` in one of my notebooks, a white space is displayed instead of plot widget.
```
cv_data = cv(
    Pool(X, y, cat_features=None),
    model.get_params(),
    plot=True
)
```
In another notebook same code shows plot without problems.

As a workaround I downgraded catboost to version 0.8.1, now widget is shown in both notebooks.

**Comment by elfwired** [comment]
Sorry for long answering, I tested what you proposed and I'm still geting the same behavior.
I also tried clearing browser cache (Chrome)
Now in some of new notebooks, that I created after installing `catboost 0.8.1.1` I see `fit()` drawing white space instead of graph too.

Also I tried `MetricVisualizer()` while `fit is running` in another notebook, and I'm getting the following error:
```
PermissionError: [Errno 13] Permission denied: 'CatBoost_OMTL_UL_Inc_Second_Mnth__catboost_info__model1\\catboost_training.json'
```
I tried calling it with absolute path and got the same error.
